cmd/coreos-assembler.go

```diff
@@ -96,6 +96,12 @@ func run(argv []string) error {
 	case "build-extensions-container", // old alias
 		"buildextend-extensions-container":
 		return buildExtensionContainer()
+	case "build":
+		mustBuildOCI, ok := os.LookupEnv("COREOS_ASSEMBLER_BUILD_OCI")
+		if ok && mustBuildOCI == "1" {
+			return buildOCI()
```

Comment:
Nitpick : Both way build an OCI image, so the name feel a bit weird IMO. 
Maybe `build_container_native` ?
Also, it would be easier to have an alias or a flag :  `cosa build --native` or something like that

Reply:
@jbtrystram Modified. LMK if the new names are fines. Added --native as well